src/LinAlg/hiopMatrixSparseCsrCuda.cpp

```diff
@@ -773,7 +779,7 @@ void hiopMatrixSparseCSRCUDA::form_transpose_from_numeric(const hiopMatrixSparse
 
 void hiopMatrixSparseCSRCUDA::form_diag_from_symbolic(const hiopVector& D)
 {
-  int m = D.get_size();
+  const int m = D.get_size();
```

Comment:
Is this function finished? seems that the sparsity patten hasn't been assigned yet.

Reply:
it is not, that code got lost somewhere somehow ...